l10n/th-th/inclusive-planning-process.md

```diff
@@ -0,0 +1,54 @@
+  * ก่อนเริ่มกิจกรรม: 
+    * สามารถศึกษาเพิ่มเติมได้[ที่นี่](https://github.com/jennymevents/virtual-events/blob/main/virtual-event-code-of-conduct.md). 
+    * จัดวางในรายละเอียดของงาน, ไม่ใช่แค่ไว้ด้านล่างของรายละเอียดเท่านั้น. เน้นว่า Code of Conducts จะใช้กับกิจกรรมงานสัมมนา,รวมถึงกิจกรรมหลังเลิกงาน, โพสใน Social Media, และกิจกรรมอื่นๆ ของชุมชน ที่เกี่ยวข้องกับงานสัมมนาหรือผู้เข้าร่วมงาน  
+    * เพิ่มตัวอ้างอิง และตัวเลือกยอมรับเข้าไปในแบบฟอร์มลงทะเบียน
+    * เพิ่มตัวเลือกให้ผู้เข้าร่วมงาน ในแบบฟอร์มลงทะเบียน เช่น การวิธีเรียก (เช่นคำนำหน้า), ความต้องการทางโภชนาการ, ความต้องการเพิ่มเติมด้านกายภาพ และสุขภาพจิต, อาการภูมิแพ้, และตัวเลือกอื่นๆ ที่จำเป็น
+    * ขอความร่วมมือกับผู้เข้าร่วมงานเพื่อให้สอดคล้องกับเป้าหมายของ Code of Conduct
+  * ระหว่าง และหลังกิจกรรม: 
+    * จัดตั้งทีม Code of Conduct โดยให้แน่ใจว่าสมาชิกในทีมมีความหลากหลาย และเป็นตัวแทนจากแต่ละส่วน
+    * อ้างอิง Code of Conduct และทีมจัดการ Code of Conduct ด้วยข้อมูลสำหรับติดต่อได้ง่าย
+    * จัดตั้งผู้ดูแล และเจ้าหน้าที่สำหรับตรวจสอบการพบปะพูดคุย และการแชท เพื่อให้แน่ใจว่าเป็นไปตาม Code of Conduct และสามารถแก้ไข และระบุปัญหาที่เกิดขึ้นได้อย่างรวดเร็ว
+    * ระบุต่อพฤติกรรมที่ไม่เหมาะสมทันทีที่ได้รับรายงาน ควรแก้ไขปัญหาอย่างรวดเร็ว และทำการบันทึกไว้ สามารถพิจารณาประกาศให้สาธารณะรับรู้ได้ ขึ้นอยู่กับความรุนแรง, พร้อมทั้งระบุมาตรการเพื่อป้องกันเหตุการณ์ที่อาจเกินซ้ำได้ 
+
+**3) พยายามผลักดันเรื่องความเปิดกว้าง และเรื่องความแตกต่าง ตลอดงานสัมมนา**
+  * ช่วงก่อน ระหว่างงาน และหลังจบงาน 
+  * จัดเวิร์คชิป สำหรับผู้เริ่มต้น และผู้เข้าร่วมงานใหม่ ขนานไปกับตัวงานหลัก เพื่อเปิดโอกาสให้พวกเขากลายเป็นผู้เข้าร่วมงานต่อไปของเรา
+  * สนับสนุนด้านความแตกต่าง ให้ครอบคลุมการเข้าถึงตัวงานกิจกรรา เช่นค่าเข้าร่วมสัมมนา ที่อาจจะกลายเป็นอุปสรรคต่อการเข้าร่วมงานโดยไม่เจตนา การเข้าร่วมงานของผู้ที่มีความหลากหลายนี้ ควรจะอยู่ในระดับเดียวกับการเข้าร่วมงานมาตรฐาน เราสามารถสอบถาม หรือจัดหาการขนส่งที่เหมาะสมได้
+  * เราสามารถทำให้หัวข้อเรื่องความเปิดกว้าง หรือความหลากหลายนี้ชัดเจนมากขึ้นได้ จากการกำหนดหัวข้อที่เกี่ยวข้องโดยตรง เช่น, ให้มีวิทยากร[ที่สามารถบรรยายเรื่องเกี่ยวกับสุขภาพจิตได้](http://mhprompt.org/).  
+  * การเข้าถึงประสบการณ์ของงาน ต้องเข้าถึงได้สำหรับทุกคน และออกแบบเพื่อให้รองรับความแตกต่าง และหลากหลายของมนุษย์เรา  
+    * นี่รวมไปถึงการมีคำบรรยายใต้ภาพ หรือวิดีโอ การจัดสีอัตราส่วน contrast 4:5:1, ตัวควบคุมการเล่น หรือหยุด, คำบรรยายเสียง, transcripts, และส่วนอื่นๆที่เข้ากับมาตรฐาน accessibility.
+    * ถ้าเป็นการวางแผนแบบเจอหน้ากัน ให้ใช้เช็คลิสต์ในการจัดเตรียมสถานที่ และความพร้อมเพื่อสนับสนุนกิจกรรมที่เปิดกว้าง ให้แน่ใจว่าผู้เข้าร่วมงานสามารถชี้แจงความต้องการที่เจาะจงได้ เช่นความต้องการด้านโภชนาการ หรืออาการแพ้อาหาร อุปกรณ์สนับสนุนด้านกายภาพ หรือสุขภาพจิตระหว่างการร่วมงานเป็นต้น
+
+**4) หลังเสร็จสิ้นกิจกรรม ให้มองหาวิธีที่จะปรับปรุงประสบการณ์ที่ได้รับในกิจกรรม**  
+  * สอบถามความคิดเห็น จากผู้เข้าร่วมงาน และวิทยากรเกี่ยวกับประสบการณ์ที่ได้รับระหว่างกิจกรรม
+  * ทบทวน Code of Conduct เพื่อให้กำหนดขอบเขตชัดเจนถึงสิ่งที่ควร และไม่ควรทำ
```

Comment:
Code of Conduct => ควรใช้ เป็นคำว่า **แนวทางปฎิบัติ** ดีไหมครับ

Reply:
ตรงนี้ไม่แน่ใจเลย หรือเราจะวงเล็บเข้าไปด้วยแบบนี้ดีครับ 

ทบทวนแนวทางปฎิบัติ (Code of Conduct)

Reply:
พลว่าได้ล่ะ 

ในหัวข้อใหญ่ที่กล่าวถึง Code of Conduct จะใส่วงเล็บเข้าไปด้านหลังแนวทางปฏิบัติ เป็น "แนวทางปฎิบัติ (Code of Conduct)" แต่ในหัวข้อย่อยจะใช้แค่คำว่า "แนวทางปฎิบัติ" อย่างเดียวครับ